src/OpenDiffix.Core/Anonymizer.fs

```diff
@@ -3,6 +3,20 @@ module OpenDiffix.Core.Anonymizer
 open System
 open System.Security.Cryptography
 
+[<RequireQualifiedAccess>]
+type AnonymizedResult<'okType> =
```

Comment:
Types are always `PascalCase`. `type AnonymizedResult<'T>` would be good enough by convention.